app/src/main/res/layout/audio_fragment.xml

```diff
@@ -36,6 +36,20 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent">
 
+      <com.google.android.material.progressindicator.CircularProgressIndicator
+        android:id="@+id/audio_loading_bar"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:indeterminate="true"
+        android:visibility="@{viewModel.playStatusLiveData == UiAudioPlayStatus.LOADING ? View.VISIBLE : View.GONE}"
+        app:indicatorColor="@color/component_color_shared_icon_white_color"
```

Comment:
You should create a component color specific to this feature so that the usage is clear. As for specific color, white is alright.